nakedret_test.go

```diff
@@ -21,25 +53,18 @@ func runNakedret(t *testing.T, filename string, maxLength uint, expected string)
 	if err := checkNakedReturns([]string{filename}, &maxLength, false); err != nil {
 		t.Fatal(err)
 	}
-	actual := string(logBuf.Bytes())
+	actual := logBuf.String()
+	fmt.Print(actual)
```

Comment:
nit: rm print call
```suggestion
```

Reply:
just resolved your concerns. can you confirm it's all good now?